src/libs/seed.js

```diff
@@ -76,8 +76,7 @@ export default {
     isSystemPhrase: function (wordList) {
         for (let word in wordList) {
             if (seedDictionary_1.default.indexOf(wordList[word]) == -1) {
-                console.log(word)
-                return false
+                return Number(word)
```

Comment:
keep `return false` here. Add new function name `findNonSystemPhrase` to find the word. Function  name should match what it is doing.